src/foam/support/model/Ticket.js

```diff
@@ -15,7 +15,7 @@ foam.CLASS({
     ],
 
     tableColumns: [
-      'id', 'requestorEmail', 'subject', 'createdAt', 'status'
+      'id', 'requestorName', 'requestorEmail', 'subject', 'createdAt', 'status'
```

Comment:
remove the requestorName table column until I review with our designer if we want it